M2/Macaulay2/tests/normal/augmented-assignment.m2

```diff
@@ -126,7 +126,7 @@ assert Equation(x#0, 5)
 
 -- install custom method
 Bar - Bar := (x, y) -> Bar(x#0 - y#0)
-Bar -= (x, y) -> if even y#0 then x#0 = 0 else Default
```

Comment:
Ohhh wait I see why I've been confused about this. I agree that this looks strange and should be removed, even if it wasn't for the problem I ran into. I think method keys for installing a method involving operators should be consistent, and currently they're all `((1OP, =), X)` or `((2OP, =), X, Y)`, not just `(OP, X)` or `(OP, X, Y)`. I actually would prefer changing the `=` to `:=` to make it absolutely clear that this is a method installation.

Perhaps an alternative syntactic sugar here could be:
```m2
Bar -= Bar := (x,y) -> ...
```

Reply:
All the `((1OP, =), X)` and `((2OP, =), X, Y)` methods actually involve the `=` operator, though, e.g., `((+, =), X, Y)` tells you what to do when there's something like `x + y = z`, where `x` and `y` are instances of `X` and `Y`, respectively.    Note also that we don't worry about the type of `z` when installing the method.